cloudbuild.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 7200s # 2 hours
+steps:
+  - name: gcr.io/cloud-builders/docker
+    args: ["build", "-t", "gcr.io/cloud-devrel-public-resources/storage-testbench:$TAG_NAME", "."]
+    dir: .
+    id: storage-testbench-build
+    waitFor: ["-"]
```

Comment:
That is redundant for the first step?

Reply:
I tested without it and things worked, it seems so. I didn't know that, I copied most of the structure of this file from another build which had already been setup and was running.